packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -566,11 +566,11 @@ class CircularProgressIndicator extends ProgressIndicator {
   }) : _indicatorType = _ActivityIndicatorType.material;
 
   /// Creates an adaptive progress indicator that is a
-  /// [CupertinoActivityIndicator] in iOS and [CircularProgressIndicator] in
-  /// material theme/non-iOS.
+  /// [CupertinoActivityIndicator] in iOS & macOS and [CircularProgressIndicator] in
+  /// material theme/non-Apple platforms.
   ///
   /// The [value], [valueColor], [strokeWidth], [semanticsLabel], and
-  /// [semanticsValue] will be ignored in iOS.
+  /// [semanticsValue] will be ignored in iOS & macOS.
```

Comment:
Can you update these to use TargetPlatform.iOS and TargetPlatform.macOS?

Reply:
> Can you update these to use TargetPlatform.iOS and TargetPlatform.macOS?

Where exactly? It's already checking the target platform, but the docs were a little bit outdated, and that's the goal of this PR, I might not understand what you mean. Can you explain?

Reply:
I was referring to the doc references. These turn into links when rendered in our documentation.